app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx

```diff
@@ -56,29 +55,37 @@ export class SauceLabsFormFields extends Component {
     disabled: PropTypes.bool,
     lineAlign: PropTypes.bool,
     initialData: PropTypes.object,
+    pluginDetails: PropTypes.object,
   };
 
   static defaultProps = {
     disabled: false,
     lineAlign: false,
-    initialData: {
-      dataCenter: DEFAULT_DATA_CENTER,
+    initialData: {},
+    pluginDetails: {
+      dataCenters: [],
     },
   };
 
   constructor(props) {
     super(props);
     const {
-      intl: { formatMessage },
+      pluginDetails: { dataCenters = [] },
     } = props;
-    this.dataCenterOptions = [
-      { value: DATA_CENTER_US, label: formatMessage(messages.dataCenterUS) },
```

Comment:
Did you forget to remove corresponding messages?

Reply:
Removed

Reply:
Fixed